### ISSUE TITLE
Blades in the Dark Clock

### ISSUE THREAD

**Issue by brunocalado** [Larkinabout/fvtt-challenge-tracker#24]
Hi.

Is it possible to achieve a clock like that:
![image](https://user-images.githubusercontent.com/4999783/189464108-a1209b0f-e85f-4f8b-9142-af3d318dab94.png)

This style is used in Blades in the Dark system.

**Comment by Larkinabout** [comment]
If you're looking for the hand-drawn look, you could use a background image with this code:

`ChallengeTracker.open({size: 250, outerTotal: 4, frameWidth: 'none', backgroundImage: 'https://.../blades-in-the-dark-4.png', outerColor: '#000000ff', outerBackgroundColor: '#00000000'})`

A better artist than me would need to  create the images, but I've made a rough attempt:
![blades-in-the-dark-4](https://user-images.githubusercontent.com/105953297/189475442-ce21fcb3-df94-4a78-87ff-71266e115ab6.png)

The challenge tracker will look like this:
![blades-in-the-dark-example](https://user-images.githubusercontent.com/105953297/189475524-d676a9a1-bfc0-46e2-9505-c2b975784519.png)

If someone does design some really good images, I'd be happy to add them to the module for other users.

**Comment by Larkinabout** [comment]
You could also just use two images with `frameWidth: 'none'`: a background image with the clock unfilled and a set image with the clock fully filled. That'd then work with any sort of transparency. As long as you include the divider/frame in both images, it will look like the clock is being filled in as you click.

**Comment by brunocalado** [comment]
Thank you. I'll try it out!

**Comment by brunocalado** [comment]
```js
const img = 'modules/bloodstone/assets/clocks/Progress%20Clock%204-0.svg'

ChallengeTracker.open({
  size: 250, 
  outerTotal: 4, 
  frameWidth: 'none', 
  backgroundImage: img , 
  outerColor: '#000000ff', 
  outerBackgroundColor: '#00000000',
  frameWidth: 'none'
});
```

error: 
![image](https://user-images.githubusercontent.com/4999783/189505859-11f1955a-5930-4bb1-bb25-4c70321d4d77.png)
[clocks.zip](https://github.com/Larkinabout/fvtt-challenge-tracker/files/9541569/clocks.zip)

**Comment by Larkinabout** [comment]
Try testing with a raster file format (png or webp) as I'm not sure it'll handle vector files. If it's that, I'll probably need to let users know somewhere that it's only raster file formats.

**Comment by Larkinabout** [comment]
Fixed in release 2.1.
